spec/memfs/dir_spec.rb

```diff
@@ -54,6 +54,23 @@ module MemFs
       end
     end
 
+    if Gem::Requirement.new('>= 2.6').satisfied_by?(Gem::Version.new(RUBY_VERSION))
+      describe '.children' do
+        it 'returns an array containing all of the filenames except for "." and ".."'\
+            'in this directory.' do
```

Comment:
Don't worry about line length for specs descriptions. I excluded the spec files from rubocop to avoid having this weird cuts 😄